frontend/components/AddHostsModal/PlatformWrapper/PlatformWrapper.tsx

```diff
@@ -263,7 +263,7 @@ const PlatformWrapper = ({
             >
               Fleet command-line tool
             </a>{" "}
-            installed:
+            installed to generate an install package:
```

Comment:
We decided to cut the text after the link, to keep this short:
```suggestion
            :
```